### ISSUE TITLE
Mod not working

### ISSUE THREAD

**Issue by StealtHiBye** [sharkster91/ExpandedHelicopterEvents#6]
Tried a few times with some of the mods I usually use, but it didn't work. So, I created a new game with only this and cheat menu mods enabled (also set helicopter event to never in sandbox options, apocalypse settings; I did tried a few times with it enabled, but the issue remained). Activated ghost and god mode while searching for radio with emergency frequency. The radio did say there was air activity, but nothing happened throught the day. A new error pop up occasionally at new hours (11h00m, 13h00m, and so on), which I'm guessing is when the events happens. Waited until radio stop informing air activity and exited the game. Here's the error log.
[console.txt](https://github.com/sharkster91/ExpandedHelicopterEvents/files/6659560/console.txt)

**Comment by ditoseadio** [comment]
Same for me.
I can't get this mod to work in any way.

**Comment by ChuckTheSheep** [comment]
Let me know if the latest workshop release fixes your issues.

**Comment by StealtHiBye** [comment]
So, did the same as before (only this and cheat menu mods enable, sandbox apocalypse settings with helicopter set to never, didn't change anything in the options) and still got only error messages (4 this time, at new hours, and the EABS showed 2 air traffic detected messages, but nothing showed up). Here's the log. Also, there was one test I did before with other mods enabled, which I played for around 16-17 days, and the only moment I had the air traffic message was in the first day. It didn't come up at later days, almost like the mod was disable after the first day.
[console.txt](https://github.com/sharkster91/ExpandedHelicopterEvents/files/6679155/console.txt)
